src/plugins/data_source/config.ts

```diff
@@ -32,6 +33,9 @@ export const configSchema = schema.object({
   clientPool: schema.object({
     size: schema.number({ defaultValue: 5 }),
   }),
+  audit: schema.object({
```

Comment:
I think we should have a config item to enable or disable audit log, because user may not want it

Reply:
make sense to me, added